firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -715,6 +716,19 @@ class BrowserCoordinator: BaseCoordinator,
         present(navigationController)
     }
 
+    func showNativeErrorPage(overlayManager: OverlayModeManager) {
+        let errorPageModel = ErrorPageModel(errorTitle: "", errorDecription: "", errorCode: "")
```

Comment:
do you want to set + pass the hardcoded strings in here instead of the VC directly?